src/vm/instruction_set.md

```diff
@@ -1784,81 +1784,83 @@ Set `$rA` to the index of the currently-verifying predicate.
+| `GTF_CREATE_MATURITY`                      | `0x012` | `tx.maturity`                                       |
+| `GTF_CREATE_BYTECODE_LENGTH`               | `0x013` | `tx.bytecodeLength`                                 |
+| `GTF_CREATE_BYTECODE_WITNESS_INDEX`        | `0x014` | `tx.bytecodeWitnessIndex`                           |
+| `GTF_CREATE_STORAGE_SLOTS_COUNT`           | `0x015` | `tx.storageSlotsCount`                              |
+| `GTF_CREATE_INPUTS_COUNT`                  | `0x016` | `tx.inputsCount`                                    |
+| `GTF_CREATE_OUTPUTS_COUNT`                 | `0x017` | `tx.outputsCount`                                   |
+| `GTF_CREATE_WITNESSES_COUNT`               | `0x018` | `tx.witnessesCount`                                 |
+| `GTF_CREATE_SALT`                          | `0x019` | Memory address of `tx.salt`                         |
+| `GTF_CREATE_STORAGE_SLOT_AT_INDEX`         | `0x01A` | Memory address of `tx.storageSlots[$rB]`            |
+| `GTF_CREATE_INPUT_AT_INDEX`                | `0x01B` | Memory address of `tx.inputs[$rB]`                  |
+| `GTF_CREATE_OUTPUT_AT_INDEX`               | `0x01C` | Memory address of `t.outputs[$rB]`                  |
+| `GTF_CREATE_WITNESS_AT_INDEX`              | `0x01D` | Memory address of `tx.witnesses[$rB]`               |
+| `GTF_INPUT_TYPE`                           | `0x101` | `tx.inputs[$rB].type`                               |
+| `GTF_INPUT_COIN_TX_ID`                     | `0x102` | Memory address of `tx.inputs[$rB].txID`             |
+| `GTF_INPUT_COIN_OUTPUT_INDEX`              | `0x103` | `tx.inputs[$rB].outputIndex`                        |
+| `GTF_INPUT_COIN_OWNER`                     | `0x104` | Memory address of `tx.inputs[$rB].owner`            |
+| `GTF_INPUT_COIN_AMOUNT`                    | `0x105` | `tx.inputs[$rB].amount`                             |
+| `GTF_INPUT_COIN_ASSET_ID`                  | `0x106` | Memory address of `tx.inputs[$rB].asset_id`         |
+| `GTF_INPUT_COIN_TX_POINTER`                | `0x107` | Memory address of `tx.inputs[$rB].txPointer`        |
+| `GTF_INPUT_COIN_WITNESS_INDEX`             | `0x108` | `tx.inputs[$rB].witnessIndex`                       |
+| `GTF_INPUT_COIN_MATURITY`                  | `0x109` | `tx.inputs[$rB].maturity`                           |
+| `GTF_INPUT_COIN_PREDICATE_LENGTH`          | `0x10A` | `tx.inputs[$rB].predicateLength`                    |
+| `GTF_INPUT_COIN_PREDICATE_DATA_LENGTH`     | `0x10B` | `tx.inputs[$rB].predicateDataLength`                |
+| `GTF_INPUT_COIN_PREDICATE`                 | `0x10C` | Memory address of `tx.inputs[$rB].predicate`        |
+| `GTF_INPUT_COIN_PREDICATE_DATA`            | `0x10D` | Memory address of `tx.inputs[$rB].predicateData`    |
+| `GTF_INPUT_COIN_PREDICATE_GAS_USED`        | `0xXXX` | `tx.inputs[$rB].predicate.gasUsed`                  |
```

Comment:
```suggestion
```

We may not need GTF on this field since it is malleable and might change between predicate estimation and verification.

Reply:
Hmmm. There are GTF for other malleable fields. It should either be consistently that such fields have a GTF, or not.